src/wp-admin/includes/plugin-install.php

```diff
@@ -399,6 +399,7 @@ function display_plugins_table() {
 			);
 			break;
 		case 'install_plugins_featured':
+			echo '<br />';
```

Comment:
Might as well remove the self-closing tag. It serves no purpose anymore (and it never really did, in practice).

```suggestion
			echo '<br>';
```

Reply:
The coding standards document still [requires closed tags](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/html/#self-closing-elements). I wouldn't mind changing that, but not only for this one instance.

Reply:
The coding standards are linking off to the XHTML spec, but WordPress does not actually serve XHTML. So yeah, I think the standards should be changed. I know @dmsnell is keen about this as well.

Reply:
@westonruter I would love to update those docs.

@sabernhardt Unfortunately, like most of the internet, the coding standards is probably a bit confused on what exactly the self-closing flag is and what HTML is and we’re enforcing misunderstandings about the relationship between XML and HTML. There are wildly more significant problems with treating HTML as XML (e.g. script and style escaping).

For those curious, I [wrote about this and its history](https://fluffyandflakey.blog/2024/11/14/five-common-html-myths/#self-closing-flag).